2024-06-18 14:18:01,678 INFO    StreamThr :57766 [internal.py:wandb_internal():85] W&B internal server running at pid: 57766, started at: 2024-06-18 14:18:01.678021
2024-06-18 14:18:01,679 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status
2024-06-18 14:18:01,682 INFO    WriterThread:57766 [datastore.py:open_for_write():87] open: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/run-cse5omnk.wandb
2024-06-18 14:18:01,682 DEBUG   SenderThread:57766 [sender.py:send():379] send: header
2024-06-18 14:18:01,699 DEBUG   SenderThread:57766 [sender.py:send():379] send: run
2024-06-18 14:18:02,208 INFO    SenderThread:57766 [dir_watcher.py:__init__():211] watching files in: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files
2024-06-18 14:18:02,209 INFO    SenderThread:57766 [sender.py:_start_run_threads():1188] run started: cse5omnk with start time 1718713081.67881
2024-06-18 14:18:02,222 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: check_version
2024-06-18 14:18:02,222 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: check_version
2024-06-18 14:18:02,339 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: run_start
2024-06-18 14:18:02,348 DEBUG   HandlerThread:57766 [system_info.py:__init__():26] System info init
2024-06-18 14:18:02,348 DEBUG   HandlerThread:57766 [system_info.py:__init__():41] System info init done
2024-06-18 14:18:02,348 INFO    HandlerThread:57766 [system_monitor.py:start():194] Starting system monitor
2024-06-18 14:18:02,348 INFO    SystemMonitor:57766 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-06-18 14:18:02,349 INFO    HandlerThread:57766 [system_monitor.py:probe():214] Collecting system info
2024-06-18 14:18:02,349 INFO    SystemMonitor:57766 [interfaces.py:start():188] Started cpu monitoring
2024-06-18 14:18:02,350 INFO    SystemMonitor:57766 [interfaces.py:start():188] Started disk monitoring
2024-06-18 14:18:02,354 INFO    SystemMonitor:57766 [interfaces.py:start():188] Started gpuapple monitoring
2024-06-18 14:18:02,358 INFO    SystemMonitor:57766 [interfaces.py:start():188] Started memory monitoring
2024-06-18 14:18:02,359 INFO    SystemMonitor:57766 [interfaces.py:start():188] Started network monitoring
2024-06-18 14:18:02,454 ERROR   HandlerThread:57766 [gpu_apple.py:probe():176] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 164, in probe
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:02,456 DEBUG   HandlerThread:57766 [system_info.py:probe():152] Probing system
2024-06-18 14:18:02,461 DEBUG   HandlerThread:57766 [system_info.py:_probe_git():137] Probing git
2024-06-18 14:18:02,474 DEBUG   HandlerThread:57766 [system_info.py:_probe_git():145] Probing git done
2024-06-18 14:18:02,474 DEBUG   HandlerThread:57766 [system_info.py:probe():200] Probing system done
2024-06-18 14:18:02,474 DEBUG   HandlerThread:57766 [system_monitor.py:probe():223] {'os': 'macOS-14.0-arm64-arm-64bit', 'python': '3.11.4', 'heartbeatAt': '2024-06-18T12:18:02.456866', 'startedAt': '2024-06-18T12:18:01.672116', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/Users/philip/Desktop/Bachelor thesis project/transformer_train.py', 'codePathLocal': 'transformer_train.py', 'codePath': 'transformer_train.py', 'git': {'remote': 'https://github.com/Rockyph/Bachelor-thesis-project.git', 'commit': '88d29ba1325adde1e67752ecb38bc6337406ce9a'}, 'email': 'philipchristianrocki@gmail.com', 'root': '/Users/philip/Desktop/Bachelor thesis project', 'host': 'Philip.local', 'username': 'philip', 'executable': '/usr/local/bin/python3', 'cpu_count': 8, 'cpu_count_logical': 8, 'disk': {'/': {'total': 460.4317207336426, 'used': 13.794960021972656}}, 'gpuapple': {'type': 'arm', 'vendor': 'Apple'}, 'memory': {'total': 16.0}}
2024-06-18 14:18:02,474 INFO    HandlerThread:57766 [system_monitor.py:probe():224] Finished collecting system info
2024-06-18 14:18:02,474 INFO    HandlerThread:57766 [system_monitor.py:probe():227] Publishing system info
2024-06-18 14:18:02,475 INFO    HandlerThread:57766 [system_monitor.py:probe():229] Finished publishing system info
2024-06-18 14:18:02,476 DEBUG   SenderThread:57766 [sender.py:send():379] send: files
2024-06-18 14:18:02,476 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2024-06-18 14:18:02,530 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:02,538 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: python_packages
2024-06-18 14:18:02,539 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: python_packages
2024-06-18 14:18:02,539 DEBUG   SenderThread:57766 [sender.py:send():379] send: telemetry
2024-06-18 14:18:02,539 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:02,539 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: stop_status
2024-06-18 14:18:02,539 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: stop_status
2024-06-18 14:18:03,214 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-metadata.json
2024-06-18 14:18:03,214 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:18:03,214 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/requirements.txt
2024-06-18 14:18:03,222 INFO    wandb-upload_0:57766 [upload_job.py:push():130] Uploaded file /var/folders/fk/p554nvt53n7d415kgm1817100000gn/T/tmpwao5bvtswandb/5vtcj40u-wandb-metadata.json
2024-06-18 14:18:03,547 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:03,631 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:03,632 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:03,632 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:03,632 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:04,215 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:04,252 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:04,253 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:04,253 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:04,255 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:04,553 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:04,660 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:04,868 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:04,868 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:04,868 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:04,869 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:05,221 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:05,221 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:18:05,471 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:05,471 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:05,471 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:05,471 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:05,557 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:06,066 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:06,066 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:06,067 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:06,067 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:06,226 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:06,560 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:06,659 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:06,659 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:06,659 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:06,661 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:06,768 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:07,231 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:07,231 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:18:07,263 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:07,263 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:07,263 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:07,263 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:18:07,264 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:07,561 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:07,870 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:07,870 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:07,871 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:07,871 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:08,237 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:08,465 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:08,466 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:08,466 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:08,466 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:08,565 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:08,890 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:09,077 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:09,077 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:09,077 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:09,078 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:09,242 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:09,242 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:18:09,571 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:09,680 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:09,680 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:09,680 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:09,681 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:10,247 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:10,576 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:11,029 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:11,253 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:18:11,577 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:12,258 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:18:12,549 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:18:12,582 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:13,137 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:13,587 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:14,592 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:15,247 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:15,594 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:16,596 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:17,356 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:17,544 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: stop_status
2024-06-18 14:18:17,544 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: stop_status
2024-06-18 14:18:17,599 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:17,746 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:18:18,601 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:19,466 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:19,604 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:20,606 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:21,578 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:21,608 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:22,610 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:23,565 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:18:23,615 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:23,684 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:24,620 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:25,624 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:25,791 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:26,625 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:27,630 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:27,900 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:28,567 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:18:28,635 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:29,637 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:30,006 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:30,639 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:31,641 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:32,112 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:32,549 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: stop_status
2024-06-18 14:18:32,549 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: stop_status
2024-06-18 14:18:32,643 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:33,648 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:33,761 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:18:34,221 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:34,365 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/config.yaml
2024-06-18 14:18:34,650 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:35,655 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:36,330 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:36,658 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:37,663 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:38,436 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:38,666 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:39,583 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:18:39,667 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:40,543 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:40,672 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:41,675 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:42,654 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:42,680 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:43,685 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:44,589 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:18:44,687 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:44,760 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:45,688 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:46,693 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:46,871 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:47,554 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: stop_status
2024-06-18 14:18:47,555 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: stop_status
2024-06-18 14:18:47,694 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:48,699 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:48,978 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:49,705 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:49,725 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:18:50,710 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:51,084 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:51,711 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:52,717 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:53,191 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:53,722 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:53,896 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:53,896 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:53,896 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:53,898 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:54,466 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:54,485 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:54,485 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:54,485 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:54,486 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:54,726 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:55,084 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:55,085 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:55,085 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:55,085 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:18:55,085 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:55,306 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:55,471 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:55,472 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:18:55,702 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:55,703 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:55,703 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:55,703 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:55,731 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:56,298 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:56,298 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:56,299 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:56,299 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:56,477 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:56,477 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:18:56,734 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:56,883 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:56,883 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:56,883 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:56,883 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:57,427 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:57,482 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:57,482 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:18:57,494 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:57,494 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:57,494 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:57,495 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:57,737 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:58,078 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:58,078 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:58,078 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:58,078 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:58,487 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:58,672 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:58,672 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:58,673 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:58,673 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:58,743 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:59,271 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:59,271 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:59,271 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:59,272 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:18:59,489 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:18:59,489 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:18:59,543 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:18:59,743 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:18:59,875 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: partial_history
2024-06-18 14:18:59,875 DEBUG   SenderThread:57766 [sender.py:send():379] send: history
2024-06-18 14:18:59,875 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: summary_record
2024-06-18 14:18:59,876 INFO    SenderThread:57766 [sender.py:_save_file():1454] saving file wandb-summary.json with policy end
2024-06-18 14:19:00,494 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/wandb-summary.json
2024-06-18 14:19:00,748 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:19:00,886 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:19:01,499 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:19:01,653 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:19:01,750 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:19:02,364 DEBUG   SystemMonitor:57766 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-06-18 14:19:02,365 DEBUG   SenderThread:57766 [sender.py:send():379] send: telemetry
2024-06-18 14:19:02,365 DEBUG   SenderThread:57766 [sender.py:send():379] send: stats
2024-06-18 14:19:02,505 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/output.log
2024-06-18 14:19:02,558 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: stop_status
2024-06-18 14:19:02,559 DEBUG   SenderThread:57766 [sender.py:send_request():406] send_request: stop_status
2024-06-18 14:19:02,751 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:19:03,757 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:19:03,760 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:19:04,756 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:19:05,761 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:19:05,864 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:19:06,766 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:19:06,793 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: status_report
2024-06-18 14:19:07,524 INFO    Thread-12 :57766 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/philip/Desktop/Bachelor thesis project/wandb/run-20240618_141801-cse5omnk/files/config.yaml
2024-06-18 14:19:07,771 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:19:07,975 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:19:08,773 DEBUG   HandlerThread:57766 [handler.py:handle_request():158] handle_request: internal_messages
2024-06-18 14:19:10,104 ERROR   gpuapple  :57766 [gpu_apple.py:sample():110] GPU stats error: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-06-18 14:19:10,515 INFO    cpu       :57766 [interfaces.py:monitor():138] Process cpu has exited.
2024-06-18 14:19:10,516 DEBUG   SystemMonitor:57766 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-06-18 14:19:10,517 DEBUG   SystemMonitor:57766 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-06-18 14:19:10,520 DEBUG   SenderThread:57766 [sender.py:send():379] send: stats
2024-06-18 14:19:10,642 INFO    MainThread:57766 [internal.py:handle_exit():75] Internal process exited
